src/test/java/net/oauth/jsontoken/AsyncJsonTokenParserTest.java

```diff
@@ -1,16 +1,14 @@
 package net.oauth.jsontoken;
 
-import static org.junit.Assert.assertThrows;
-
 import com.google.common.util.concurrent.Futures;
-import com.google.common.util.concurrent.ListeningExecutorService;
 import com.google.common.util.concurrent.MoreExecutors;
 import com.google.gson.JsonParseException;
+import java.security.SignatureException;
+import java.util.concurrent.*;
```

Comment:
nit: no wildcard imports: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports, here and in other files

Reply:
Fixed here and other modified files.